src/vs/workbench/contrib/logs/common/logsActions.ts

```diff
@@ -66,12 +66,12 @@ export class SetLogLevelAction extends Action {
 		const entries: (LogLevelQuickPickItem | LogChannelQuickPickItem | IQuickPickSeparator)[] = [];
 		entries.push({ type: 'separator', label: nls.localize('all', "All") });
 		entries.push(...this.getLogLevelEntries(defaultLogLevels.default, this.loggerService.getLogLevel()));
-		entries.push({ type: 'separator', label: nls.localize('loggers', "Logs") });
-		entries.push(...logs.sort((a, b) => a.label.localeCompare(b.label)));
-		if (extensionLogs.length && logs.length) {
+		if (extensionLogs.length) {
 			entries.push({ type: 'separator', label: nls.localize('extensionLogs', "Extension Logs") });
+			entries.push(...extensionLogs.sort((a, b) => a.label.localeCompare(b.label)));
```

Comment:
How many entries is `extensionLogs` going to have? Maybe you shouldn't use `...` here, as it quickly causes a stack overflow for too long lists.

Reply:
Not many. So this is safe!